src/runtime_src/core/pcie/windows/alveo/device_windows.cpp

```diff
@@ -1305,6 +1305,43 @@ struct accel_deadlock_status
   }
 };
 
+struct pcie_device_shutdown
```

Comment:
These functions needs to be implemented in windows but for now its no-op, if I don't define this query it will throw not supported error which will hinder SC firmware update in windows